.ci/windows/tools.ps1

```diff
@@ -8,7 +8,8 @@ Install-WindowsFeature -Name Web-Server, Web-Mgmt-Tools ;
 Add-WindowsFeature NET-Framework-45-ASPNET ;
 Add-WindowsFeature Web-Asp-Net45 ;
 
-# Install .Net SDK'
+# Install .Net SDKs
+& choco install dotnetcore-sdk -m -y --no-progress -r --version 3.1.100
 & choco install dotnetcore-sdk -m -y --no-progress -r --version 2.1.505
```

Comment:
Do we still need this version (` 2.1.505`) given we are installing `3.1.100`

Reply:
I'm unsure on this one. When I only had 3.1 SDK installed I saw failing builds on Linux telling that 2.1 SDK is missing and some nuget pacakges can't be found - adding 2.1 SDK solved the problem there. 

Then I tested on a clean windows machine, in that case, only with 3.1 SDK the `netcoreapp2.2` builds were skipped telling that some 2.2 stuff is missing. 

We can remove and give it a try, but as it seems, somehow only installing 3.1 is not enough.

Reply:
@v1v works in CI in #646  and we saw this:

```
2019-12-10T13:07:49.913Z] A total of 1 test files matched the specified pattern.
[2019-12-10T13:07:50.175Z] Testhost process exited with error: It was not possible to find any compatible framework version
[2019-12-10T13:07:50.175Z] The framework 'Microsoft.AspNetCore.App', version '2.2.0' was not found.
[2019-12-10T13:07:50.175Z]   - The following frameworks were found:
[2019-12-10T13:07:50.175Z]       2.1.9 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
[2019-12-10T13:07:50.175Z]       2.1.13 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
[2019-12-10T13:07:50.175Z]       3.1.0 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
[2019-12-10T13:07:50.175Z] You can resolve the problem by installing the specified framework and/or SDK.
[2019-12-10T13:07:50.175Z] The specified framework can be found at:
[2019-12-10T13:07:50.175Z]   - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=2.2.0&arch=x64&rid=win10-x64
[2019-12-10T13:07:50.175Z] . Please check the diagnostic logs for more information.
[2019-12-10T13:07:50.175Z] Results File: C:\Users\jenkins\workspace\tnet_apm-agent-dotnet-mbp_PR-646\apm-agent-dotnet\test\Elastic.Apm.AspNetCore.Tests\TestResults.xml
[2019-12-10T13:07:50.175Z] 
[2019-12-10T13:07:50.175Z] Test Run Aborted.
[2019-12-10T13:07:50.175Z
```

Reply:
Aye netcoreapp needs the sdks, somewhere I knew this and forgot :)